src/PowerShellEditorServices.Host/LanguageServer.cs

```diff
@@ -152,6 +153,46 @@ public void Initialize()
             await requestContext.SendResult(null);
         }
 
+        private async Task HandleExpandAliasRequest(
+            string content,
+            EditorSession editorSession,
+            RequestContext<object, object> requestContext)
+        {
+            var psCommand = new PSCommand();
+            var script = @"
+function Expand-Alias {
+
+    param($targetScript)
+
+    [ref]$errors=$null
+    
+    $tokens = [System.Management.Automation.PsParser]::Tokenize($targetScript, $errors).Where({$_.type -eq 'command'}) | 
+                    Sort Start -Descending
+
+    foreach ($token in  $tokens) {
+        $definition=(Get-Command ('`'+$token.Content) -CommandType Alias -ErrorAction SilentlyContinue).Definition
+
+        if($definition) {        
+            $lhs=$targetScript.Substring(0, $token.Start)
+            $rhs=$targetScript.Substring($token.Start + $token.Length)
+            
+            $targetScript=$lhs + $definition + $rhs
+       }
+    }
+
+    $targetScript
+}
+M
+Expand-Alias @'" + "\r\n" + content + "\r\n'@";
```

Comment:
This is a security risk since the content string is being inserted and executed directly as a script.  I think it might be safer if you take the "Expand-Alias" call out of the script string and then add another command to the psCommand to call it.  Something like this:

```csharp
psCommand
    .AddScript(script)
    .AddCommand("Expand-Alias")
    .AddArgument(@'" + "\r\n" + content + "\r\n'@");
```